ouroboros-network/test/Test/Ouroboros/Network/PeerSelection/MockEnvironment.hs

```diff
@@ -57,11 +57,11 @@ import           Ouroboros.Network.PeerSelection.Governor hiding
                      (PeerSelectionState (..))
 import qualified Ouroboros.Network.PeerSelection.LocalRootPeers as LocalRootPeers
 
-import           Ouroboros.Network.Testing.Data.Script (PickScript,
-                     ScriptDelay (..), TimedScript, arbitraryPickScript,
-                     initScript', interpretPickScript, playTimedScript,
-                     prop_shrink_Script, singletonScript, stepScript,
-                     stepScriptSTM)
+import           Ouroboros.Network.Testing.Data.Script (NonEmpty (..),
+                     PickScript, Script (..), ScriptDelay (..), TimedScript,
+                     arbitraryPickScript, arbitraryScriptOf, initScript',
+                     interpretPickScript, playTimedScript, prop_shrink_Script,
+                     singletonScript, stepScript)
```

Comment:
:thinking: why we need to import `NonEmpty`? In this module we only add `retry` in one of the records.

Reply:
I don't know, maybe I forgot to clean this import.